sync-core/src/test/java/com/cloudant/sync/replication/CompactedDBReplicationTest.java

```diff
@@ -81,7 +81,7 @@ public void replicationFromCompactedDB() throws Exception{
         CouchDbInfo info = couchClient.getDbInfo(getDbName());
 
         while(info.isCompactRunning()) {
-            Thread.sleep(10);
+            //Thread.sleep() causes the tests to hang for long periods
```

Comment:
@rhyshort we have other bits of test code which use Thread.sleep(), and RetriableTask uses it. So I wonder why it works in those places and not here?